lib/CMakeFiles/3.11.0/CMakeCXXCompiler.cmake

```diff
@@ -0,0 +1,75 @@
+set(CMAKE_CXX_COMPILER "/usr/lib64/ccache/c++")
```

Comment:
All the files in the folder `lib/CmakeFiles/` have probably been generated by cmake when you did an "in-source" build. Can you please remove them?

Reply:
Tried the command 
git rm -f CMakeFiles
with and without backslash at the end, but without success. Got a message "fatal: pathspec 'CMakeFiles' did not match any files".
Got the same message when trying -rf, also with and without a backslash.